tests/AssertTest.php

```diff
@@ -497,6 +625,29 @@ public function testConvertValuesToStrings($method, $args, $exceptionMessage)
 
         call_user_func_array(array('Webmozart\Assert\Assert', $method), $args);
     }
+
+    /**
+     * @param null|int $minVersion
+     * @param bool $multibyte
+     *
+     * @return bool FALSE if test must be skipped
```

Comment:
When you run ` $this->markTestSkipped()` an "skippedException" (or something like that) is thrown. This function does not need to return anything.

Reply:
good to know!